src/objects/elements.cc

```diff
@@ -199,11 +199,11 @@ void CopyObjectToObjectElements(Isolate* isolate, FixedArrayBase from_base,
                                 FixedArrayBase to_base, ElementsKind to_kind,
                                 uint32_t to_start, int raw_copy_size) {
   ReadOnlyRoots roots(isolate);
-  DCHECK(to_base.map() != roots.fixed_cow_array_map());
+  CHECK(to_base.map() != roots.fixed_cow_array_map());
```

Comment:
These DCHECK->CHECK changes should be reverted.

Reply:
Ok